clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/input/kafka/KafkaInputUtils.java

```diff
@@ -55,6 +72,35 @@ public static VeniceProperties getConsumerProperties(JobConf config) {
     return new VeniceProperties(consumerFactoryProperties);
   }
 
+  public static KafkaValueSerializer getKafkaValueSerializer(JobConf config) {
+    KafkaValueSerializer kafkaValueSerializer = new OptimizedKafkaValueSerializer();
+    boolean isSchemaReaderEnabled = Boolean.parseBoolean(config.get(SYSTEM_SCHEMA_READER_ENABLED, "false"));
+    if (isSchemaReaderEnabled) {
```

Comment:
Let us make sure the Grid/AZ has access to prod Router to retrieve system schemas before starting using this feature.

Reply:
Yeah I have checked with grid team. It has network acl to prod router.